lib/ansible/modules/system/lvg.py

```diff
@@ -167,8 +167,9 @@ def main():
         # get pv list
         pvs_cmd = module.get_bin_path('pvs', True)
         if dev_list:
-            pvs_filter = ' || '. join(['pv_name = {0}'.format(x) for x in dev_list + module.params['pvs']])
-            pvs_filter = "--select '%s'" % pvs_filter
+            pvs_filter_pv_name = ' || '. join(['pv_name = {0}'.format(x) for x in dev_list + module.params['pvs']])
```

Comment:
I believe it's safe to drop square brackets as it creates an intermediate list this way (list comprehension) without it it'd be a generator expression which is way more memory-efficient
```suggestion
             pvs_filter_pv_name = ' || '. join('pv_name = {0}'.format(x) for x in dev_list + module.params['pvs'])
```

Reply:
or even better:
```suggestion
            import itertools  # of course this should go to the top of the module
            pvs_filter_pv_name = ' || '. join('pv_name = {0}'.format(x) for x in itertools.chain(dev_list, module.params['pvs']))
```

Reply:
Indeed, like it